service/src/main/resources/db/data/vesilahti/afterMigrate__preschool_terms.sql

```diff
@@ -5,7 +5,7 @@
 INSERT INTO preschool_term
     (id, finnish_preschool, swedish_preschool, extended_term, application_period, term_breaks)
 VALUES
-    ('4a2e558c-658d-11ed-8e91-0fa1ddc0fe70', '[2023-08-09,2024-05-31]', '[2023-08-09,2024-05-31]', '[2023-08-09,2024-05-31]', '[2022-10-01,2024-05-31]', datemultirange('[2023-10-16,2023-10-22]', '[2023-12-23,2024-01-07]', '[2024-02-26,2024-03-03]', '[2024-03-28,2024-04-01]', '[2024-05-09,2024-05-10]'))
+    ('4a2e558c-658d-11ed-8e91-0fa1ddc0fe69', '[2023-08-09,2024-05-31]', '[2023-08-09,2024-05-31]', '[2023-08-09,2024-05-31]', '[2022-10-01,2024-05-31]', datemultirange('[2023-10-16,2023-10-22]', '[2023-12-23,2024-01-07]', '[2024-02-26,2024-03-03]', '[2024-03-28,2024-04-01]', '[2024-05-09,2024-05-10]'))
```

Comment:
This is not going to work for existing environments. Either revert or add `UPDATE preschool_term SET id = '4a2e558c-658d-11ed-8e91-0fa1ddc0fe69' WHERE id = '4a2e558c-658d-11ed-8e91-0fa1ddc0fe70'` before `INSERT`.